spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/store/JdbcChannelMessageStore.java

```diff
@@ -255,6 +259,19 @@ public void setMessageRowMapper(MessageRowMapper messageRowMapper) {
 	}
 
 	/**
+	 * Allows for passing in a custom {@link MessageGroupPreparedStatementSetter}.
```

Comment:
Set/Inject a custom.

But JavaDocs on methods must definitely be in imperative.